### PR TITLE
Fix issue with Gizmo+Pivot

### PR DESCRIPTION
**Pull request by CedricGuillemet**
Tested with all gizmos + inspector, with and without pivot.
Thank you for testing @PirateJC I'm sure I missed something :)

### CODE REVIEW

File: src/Gizmos/gizmo.ts [hunk 1]
```diff
@@ -231,6 +231,21 @@ export class Gizmo implements IDisposable {
         }
     }
 
+    /**
+     * Handle position/translation when using an attached node using pivot
+     */
+    protected _handlePivot() {
+        const attachedNodeTransform = this._attachedNode as any;
```
Comment (sebavan) [inline src/Gizmos/gizmo.ts:238]: Is there any better type we could cast into ?

**Review by sebavan — APPROVED** [review]
Only a tiny comment on my side :-)

**Comment by rickfromwork** [comment]
This might be a different kind of pivot, but I know some behaviors, like the baseSixDofDragBehavior, reference the mesh pivots (calling getAbsolutePivotPoint, etc.). Would anything need to change on their end?

**Comment by CedricGuillemet** [comment]
@rickfromwork Tested ok with SixDofDragBehavior, it's not the same code path.

**Review by Popov72 — APPROVED** [review]

**Pull request merged**